### ISSUE TITLE
History key binding does not work without query

### ISSUE THREAD

**Issue by bturrubiates** [junegunn/fzf#733]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [X] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [X] Linux
    - [] Mac OS X
    - [] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [X] zsh
    - [ ] fish

Older versions of FZF allowed using the ctrl-r mapping without a query. The newest iteration gives the following error:

```sh
bturrubi at savbu-usnic-a :: query string required
```

I've traced it to commit 95b34de. 

Was this intentional?

**Comment by junegunn** [comment]
No, I can't reproduce on my machine. Which version of zsh do you have?

**Comment by bturrubiates** [comment]
> No, I can't reproduce on my machine. Which version of zsh do you have?

```sh
bturrubi at savbu-usnic-a :: zsh --version
zsh 4.3.10 (x86_64-redhat-linux-gnu)
```

**Comment by junegunn** [comment]
Hmm, 4.3.10 is way too old, it was released in 2012 which was much before fzf came into being. Please consider upgrading your zsh.

**Comment by bturrubiates** [comment]
> Hmm, 4.3.10 is way too old, it was released in 2012 which was much before fzf came into being. Please consider upgrading your zsh.

Unfortunately, it seems that 4.3.10 is the newest available on RHEL 6.5. I'll find myself a workaround.
